sunpy/image/coalignment.py

```diff
@@ -105,7 +105,8 @@ def clip_edges(data, yclips, xclips):
     """
     ny = data.shape[0]
     nx = data.shape[1]
-    return data[yclips[0].value: ny - yclips[1].value, xclips[0].value: nx - xclips[1].value]
+    return data[int(yclips[0].value): ny - int(yclips[1].value),
+                int(xclips[0].value): nx - int(xclips[1].value)]
```

Comment:
That documentation is confusing as it has nothing to do with the code.
I'll change it.

The use of the int is actually to convert to integer type, not to floor the
value (the calculation of the value is done elsewhere).  I found that if
you do

x = 1 * u.pix

then x.value is returned as a float.  The purpose of the int is to ensure
an actual integer is passed in.



On Sat, May 14, 2016 at 2:30 PM, dpshelio <notifications@github.com> wrote:

> In sunpy/image/coalignment.py
> <https://github.com/sunpy/sunpy/pull/1766#discussion_r63280102>:
>
> > @@ -105,7 +105,8 @@ def clip_edges(data, yclips, xclips):
> >      """
> >      ny = data.shape[0]
> >      nx = data.shape[1]
> > -    return data[yclips[0].value: ny - yclips[1].value, xclips[0].value: nx - xclips[1].value]
> > +    return data[int(yclips[0].value): ny - int(yclips[1].value),
> > +                int(xclips[0].value): nx - int(xclips[1].value)]
>
> The documentation says *ceiling* values, however, when using int you are
> *flooring* the value - should the documentation be updated?:
>
> >>> a = 3.6 * u.pix>>> print(int(a.value))3>>> print(np.ceil(a))<Quantity 4.0 pix>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/sunpy/sunpy/pull/1766/files/6fdda45041fc8dca34a28145969f0e6f43e4309e#r63280102>
>